src/pages/RoomInvitePage.js

```diff
@@ -0,0 +1,281 @@
+            if (!login.toLowerCase().trim() || !accountID) {
+                return;
+            }
+            invitedEmailsToAccountIDs[login] = Number(accountID);
+        });
+        Report.inviteToRoom(props.report.reportID, invitedEmailsToAccountIDs);
+        Navigation.goBack(backRoute);
+    }, [selectedOptions, backRoute, props.report, validate]);
+
+    const [policyName, shouldShowAlertPrompt] = useMemo(
+        () => [lodashGet(props.policy, 'name'), _.size(lodashGet(props.policy, 'errors', {})) > 0 || lodashGet(props.policy, 'alertMessage', '').length > 0],
+        [props.policy],
+    );
+
+    const headerMessage = useMemo(() => {
+        const searchValue = searchTerm.trim().toLowerCase();
+        if (!userToInvite && CONST.EXPENSIFY_EMAILS.includes(searchValue)) {
+            return translate('messages.errorMessageInvalidEmail');
+        }
+        if (!userToInvite && excludedUsers.includes(searchValue)) {
+            return translate('messages.userIsAlreadyMemberOfWorkspace', {login: searchValue, workspace: policyName});
+        }
+        return OptionsListUtils.getHeaderMessage(personalDetails.length !== 0, Boolean(userToInvite), searchValue);
+    }, [excludedUsers, translate, searchTerm, policyName, userToInvite, personalDetails]);
+    return (
+        <ScreenWrapper
+            shouldEnableMaxHeight
+            testID={RoomInvitePage.displayName}
+        >
+            {({didScreenTransitionEnd}) => {
+                const sections = didScreenTransitionEnd ? getSections() : [];
+
+                return (
+                    <FullPageNotFoundView
+                        shouldShow={_.isEmpty(props.report)}
+                        subtitleKey={_.isEmpty(props.report) ? undefined : 'roomMembersPage.notAuthorized'}
+                        onBackButtonPress={() => Navigation.goBack(backRoute)}
+                    >
+                        <HeaderWithBackButton
+                            title={translate('workspace.invite.invitePeople')}
+                            subtitle={ReportUtils.getReportName(props.report)}
+                            onBackButtonPress={() => {
+                                Navigation.goBack(backRoute);
+                            }}
+                        />
+                        <SelectionList
+                            canSelectMultiple
+                            sections={sections}
+                            textInputLabel={translate('optionsSelector.nameEmailOrPhoneNumber')}
+                            textInputValue={searchTerm}
+                            onChangeText={setSearchTerm}
+                            headerMessage={headerMessage}
+                            onSelectRow={toggleOption}
+                            onConfirm={inviteUsers}
+                            showScrollIndicator
+                            showLoadingPlaceholder={!didScreenTransitionEnd || !OptionsListUtils.isPersonalDetailsReady(props.personalDetails)}
```

Comment:
```suggestion
                            showLoadingPlaceholder={!didScreenTransitionEnd || !OptionsListUtils.isPersonalDetailsReady(props.personalDetails)}
                            shouldPreventDefaultFocusOnSelectRow={!Browser.isMobile()}
```

For consistency I think we need this prop